# FTP::Remove

string file_path
---
bool success
